QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java

```diff
@@ -18,12 +18,14 @@ public static void setupJoysticks() {
 		DriveCode.leftJoystickChannel = Igneous.joystick1.getYAxisSource();
 		DriveCode.rightJoystickChannel = Igneous.joystick1.getAxisSource(6);	
 		DriveCode.octocanumShifting = BooleanMixing.createDispatch(Igneous.joystick1.getButtonChannel(1), Igneous.globalPeriodic);
+		
+//		Elevator.raisingInput = BooleanMixing.createDispatch(FloatMixing.floatIsAtLeast(Igneous.joystick1.getAxisChannel(3), 0.9f), Igneous.globalPeriodic);
+//		Elevator.loweringInput = BooleanMixing.createDispatch(FloatMixing.floatIsAtLeast(Igneous.joystick1.getAxisChannel(4), 0.9f), Igneous.globalPeriodic);
+		Elevator.raisingInput = BooleanMixing.createDispatch(Igneous.joystick1.getButtonChannel(2), Igneous.globalPeriodic);
+		Elevator.loweringInput = BooleanMixing.createDispatch(Igneous.joystick1.getButtonChannel(3), Igneous.globalPeriodic);
```

Comment:
Perhaps you should send events instead of booleans for these two buttons. This also makes your code simpler - you can just do `Igneous.joystick1.getButtonSource(button)` and it'll give you an EventInput for that button.